### ISSUE TITLE
Localization support

### ISSUE THREAD

**Issue by jluis2k10** [petio-team/petio#267]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Petio is english only, I'd love to have the possibility of setting it in other language. Not only the texts in the interface but in the metadata pulled from TMDB as well.
From what I've seen in the code it's not even prepared for localization :( but a man can dream...

**Describe alternatives you've considered**
Well i could try to teach english to my users, not sure if that's an option.

**Additional context**
None.

**Comment by AshDyson** [comment]
Yeah as I mentioned on your other issue it's planned just not even started yet until we have final strings and stability in the prototype release.

**Comment by AshDyson** [comment]
Merging your sonarr / radarr languages request into this

**Comment by ADRFranklin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Localization Support](https://trello.com/c/dDdUaijT/333-localization-support)
